babel/dates.py

```diff
@@ -1287,8 +1287,20 @@ def format_period(self, char):
         return get_period_names(locale=self.locale)[period]
 
     def format_frac_seconds(self, num):
-        value = str(self.value.microsecond)
-        return self.format(round(float('.%s' % value), num) * 10**num, num)
+        value = self.value.microsecond / 1000000
+        return self.format(round(value, num) * 10**num, num)
+        """ Returns fractional seconds. Converts microseconds received to the \
+            nearest round-off value according to the given precesion value.
+
+            Earlier the function computes wrong result if the microseconds \
```

Comment:
This doesn't need to be in the docstring. We don't need to document that a function used to do the wrong thing.